devserver.sh

```diff
@@ -8,6 +8,14 @@ datasource_container="cryostat-devserver-jfr-datasource"
 grafana_container="cryostat-devserver-grafana-dashboard"
 podname="cryostat-devserver"
 
+getPomProperty() {
+    if command -v xpath > /dev/null 2>&1 ; then
+        xpath -q -e "project/properties/$1/text()" pom.xml
+    else
+        ${MVN} help:evaluate -o -B -q -DforceStdout -Dexpression="$1"
+    fi
+}
```

Comment:
Nice, this function turned out cleaner looking than I expected it would with just the `$1`. Makes sense.